### ISSUE TITLE
The inference process is killed.

### ISSUE THREAD

**Issue by lxxie298** [fudan-generative-vision/hallo3#37]
Thank you for your shared new work !! 
I have some problems when I used the inference script. What should I do to handle this problem?

I installed hallo3 by following the README and used xformers==0.0.27.post2
When I run inference script provided on README.md, it shows following problem:

> (hallo3) xx@gpu15:~/code/hallo3$ CUDA_VISIBLE_DEVICES='0' python hallo3/sample_video.py --base ./configs/cogvideox_5b_i2v_s2.yaml ./configs/inference.yaml --seed 21766 --input-file ./examples/inference/input.txt --output-dir ./output
> [2025-03-04 15:04:28,890] [INFO] [real_accelerator.py:203:get_accelerator] Setting ds_accelerator to cuda (auto detect)
>  [WARNING]  async_io requires the dev libaio .so object and headers but these were not found.
>  [WARNING]  async_io: please install the libaio-dev package with apt
>  [WARNING]  If libaio is already installed (perhaps from source), try setting the CFLAGS and LDFLAGS environment variables to where it can be found.
>  [WARNING]  Please specify the CUTLASS repo directory as environment variable $CUTLASS_PATH
>  [WARNING]  sparse_attn requires a torch version >= 1.5 and < 2.0 but detected 2.4
>  [WARNING]  using untested triton version (3.0.0), only 1.0.0 is known to be compatible
> /share/home/HCI/lianxin/anaconda3/envs/hallo3/lib/python3.10/site-packages/deepspeed/runtime/zero/linear.py:49: FutureWarning: `torch.cuda.amp.custom_fwd(args...)` is deprecated. Please use `torch.amp.custom_fwd(args..., device_type='cuda')` instead.
>   def forward(ctx, input, weight, bias=None):
> /share/home/HCI/lianxin/anaconda3/envs/hallo3/lib/python3.10/site-packages/deepspeed/runtime/zero/linear.py:67: FutureWarning: `torch.cuda.amp.custom_bwd(args...)` is deprecated. Please use `torch.amp.custom_bwd(args..., device_type='cuda')` instead.
>   def backward(ctx, grad_output):
> /share/home/HCI/lianxin/anaconda3/envs/hallo3/lib/python3.10/site-packages/kornia/feature/lightglue.py:44: FutureWarning: `torch.cuda.amp.custom_fwd(args...)` is deprecated. Please use `torch.amp.custom_fwd(args..., device_type='cuda')` instead.
>   @torch.cuda.amp.custom_fwd(cast_inputs=torch.float32)
> /share/home/HCI/lianxin/anaconda3/envs/hallo3/lib/python3.10/site-packages/xformers/ops/fmha/flash.py:211: FutureWarning: `torch.library.impl_abstract` was renamed to `torch.library.register_fake`. Please use that instead; we will remove `torch.library.impl_abstract` in a future version of PyTorch.
>   @torch.library.impl_abstract("xformers_flash::flash_fwd")
> /share/home/HCI/lianxin/anaconda3/envs/hallo3/lib/python3.10/site-packages/xformers/ops/fmha/flash.py:344: FutureWarning: `torch.library.impl_abstract` was renamed to `torch.library.register_fake`. Please use that instead; we will remove `torch.library.impl_abstract` in a future version of PyTorch.
>   @torch.library.impl_abstract("xformers_flash::flash_bwd")
> [2025-03-04 15:04:44,069] [WARNING] No training data specified
> [2025-03-04 15:04:44,069] [WARNING] No train_iters (recommended) or epochs specified, use default 10k iters.
> [2025-03-04 15:04:44,069] [INFO] using world size: 1
> [2025-03-04 15:04:44,081] [INFO] [RANK 0] > initializing model parallel with size 1
> [2025-03-04 15:04:44,082] [INFO] [RANK 0] You didn't pass in LOCAL_WORLD_SIZE environment variable. We use the guessed LOCAL_WORLD_SIZE=1. If this is wrong, please pass the LOCAL_WORLD_SIZE manually.
> [2025-03-04 15:04:44,090] [INFO] [RANK 0] building SATVideoDiffusionEngine model ...
> Killed

**Comment by TanSongBai** [comment]
same problem, how to solve it?

**Comment by GodLinW** [comment]
same problem, how to solve it?do you deal it？
